prototype_source/flight_recorder_tutorial.rst

```diff
@@ -48,8 +48,6 @@ Enabling Flight Recorder
 ------------------------
 There are two required environment variables to get the initial version of Flight Recorder working.
```

Comment:
Oh I forgot to change this.. Let make it three, I think this is indeed needed and is not optional?